frontend/src/store/store.ts

```diff
@@ -1381,7 +1396,7 @@ export default new Vuex.Store<IState>({
         state.contracts().CryptoBlades!,
         state.contracts().SkillToken,
         state.defaultAccount,
-        state.skillRewards,
+        '0',
```

Comment:
Should it be 0 here?

Reply:
Yes, so front-end asks for approval for full amount, not considering unclaimed rewards.